danielMunteanu/src/main/java/Item.java

```diff
@@ -0,0 +1,66 @@
+public class Item {
+    public long id;
+    String title;
+    Type type;
+    static long idCounter;
+
+    public Item() {
+    }
+
+    public Item(String title, Type type) {
+        this.id = ++idCounter;
+        this.title = title;
+        this.type = type;
+    }
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
+        this.id = id;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public Type getType() {
+        return type;
+    }
+
+    public void setType(Type type) {
+        this.type = type;
+    }
+
+    public static long getIdCounter() {
+        return idCounter;
+    }
+
+    public static void setIdCounter(long idCounter) {
+        Item.idCounter = idCounter;
+    }
+
+
+    @Override
+    public String toString() {
+        return "Item{" +
+                "id=" + id +
+                ", title='" + title + '\'' +
+                ", type=" + type +
+                '}';
+    }
+}
+
+
+enum Type {
```

Comment:
I would suggest to define it as a separate file, as your colleague. This is done in order to have a better view on project structure.

Reply:
done